File: 2025-1/sesion-de-clase-04/PUCP/Edu/Pe/Inf30/Sesion04/Entidades/Producto.cs
using System;

namespace PUCP.Edu.Pe.Inf30.Sesion04.Entidades {
	public class Producto : EntidadBase {
		public int Stock { get; set; }
		public double Precio { get; set; }
		
		public override void imprimir() {
			base.imprimir();
			Console.WriteLine("Stock: " + this.Stock);
			Console.WriteLine("Precio: " + this.Precio);
		}
	}
}

File: 2025-1/sesion-de-clase-04/pe/edu/pucp/inf30/sesion04/entidades/Producto.java
package pe.edu.pucp.inf30.sesion04.entidades;

public class Producto extends EntidadBase {
	private int stock;
	private double precio;
	
	public int getStock() {
		return this.stock;
	}
	
	public double getPrecio() {
		return this.precio;
	}
	
	public void setStock(int stock) {
		this.stock = stock;
	}
	
	public void setPrecio(double precio) {
		this.precio = precio;
	}
	
	@Override
	public void imprimir() {
		super.imprimir();
		System.out.println("Stock: " + this.stock);
		System.out.println("Precio: " + this.precio);
	}
}